### ISSUE TITLE
HOT RELOAD (flet==0.10.3)

### ISSUE THREAD

**Issue by itsmeadarsh2008** [flet-dev/flet#1975]
**Description**
I'm experiencing a problem with **Hot Reload** with the latest version (`flet==0.10.3`)

I think we need to change the **timeout** i.e. delay while the watchdog reloads the app when the app file is changed (the time interval). I think it's a `watchdog` bug. Try degrading the current watchdog version.
**Code example to reproduce the issue:**

```python
import flet as ft
import random


def main(page: ft.Page):
    page.title = "TrainBrain"
    page.window_width = 500
    page.vertical_alignment = ft.MainAxisAlignment.CENTER
    page.add(
        ft.Row(
            [
                ft.ElevatedButton(random.randint(-23, 115)),
                ft.ElevatedButton(random.randint(-23, 115)),
                ft.ElevatedButton(random.randint(-23, 115)),
                ft.ElevatedButton(random.randint(-23, 115))
            ],
            alignment=ft.MainAxisAlignment.CENTER,
        )
    )


ft.app(target=main)
```

**Describe the results you received:**
On changing the source file it shows this kind of **hot-reload bug**.
![image](https://github.com/flet-dev/flet/assets/71959210/f1062e42-6053-408c-a389-d4d59c13bf3a)

**Describe the results you expected:**
![image](https://github.com/flet-dev/flet/assets/71959210/a10f163f-3634-4c48-ac5e-742828383e07)

**Additional information you deem important (e.g. issue happens only occasionally):**
It shows a buffer all the time.
**Flet version (`pip show flet`)**:

```
Name: flet
Version: 0.10.3
Summary: Flet for Python - easily build interactive multi-platform apps in Python
Home-page: 
Author: Appveyor Systems Inc.
Author-email: hello@flet.dev
License: Apache-2.0
Location: D:\TrainBrain\environment\Lib\site-packages
Requires: copier, flet-runtime, packaging, qrcode, watchdog, websocket-client, websockets
Required-by: 
```

**Operating system**:
Windows
(Other people are facing the same issue in Linux or maybe in Mac)
**Additional environment details:**

**Comment by FeodorFitsner** [comment]
What timeout do you mean - could you point me to a place within the code?

**Comment by itsmeadarsh2008** [comment]
@FeodorFitsner It's not about AFK. I'm referring to the time delay between code execution in `watchdog` file change

**Comment by ndonkoHenri** [comment]
Closing in favour of https://github.com/flet-dev/flet/issues/1556
